tasks/step_stop_instance.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- import_tasks: 'prepare.yml'
+  tags:
+    - cartridge-instances
+    - cartridge-replicasets
+    - cartridge-config
```

Comment:
```suggestion
  tags: cartridge-instances
```

Reply:
The same as I said in #265: I don't like the opportunity to accidentally skip prepare.